### PR TITLE
Make object initialization consistent

### LINKED ISSUE #144: Make object initializers consistent.
When instantiating objects and setting members simultaneously, it can be seen that there are two ways of doing this (sometimes in the same file:
```
var x = new Something() { Foo = 3 };
var y = new Something { Foo = 3 };
```
Both are perfectly acceptable, but mixing and matching should be avoided.

### PR DESCRIPTION
**Pull request by Seb-stian**
Closes #144

I searched through the solution with regex and fixed all encountered inconsistencies.

### CODE REVIEW

File: Obsidian.API/_Types/ChatMessage.cs [hunk 1]
```diff
@@ -45,8 +45,8 @@ public IEnumerable<ChatMessage> GetExtras()
         /// <summary>
         /// Creates a new <see cref="ChatMessage"/> object with plain text.
         /// </summary>
-        public static ChatMessage Simple(string text) => new ChatMessage() { Text = text.Replace('&', '§') };
-        public static ChatMessage Simple(string text, ChatColor color) => new ChatMessage() { Text = $"{color}{text}" };
+        public static ChatMessage Simple(string text) => new() { Text = text.Replace('&', '§') };
```
Comment (Naamloos) [inline Obsidian.API/_Types/ChatMessage.cs:48]: this uses new() { ...values }

File: Obsidian/Client.cs [hunk 2]
```diff
@@ -90,7 +90,7 @@ public Client(TcpClient tcp, Config config, int playerId, Server originServer)
             Stream parentStream = this.tcp.GetStream();
             this.minecraftStream = new MinecraftStream(parentStream);
 
-            var blockOptions = new ExecutionDataflowBlockOptions() { CancellationToken = Cancellation.Token, EnsureOrdered = true };
+            var blockOptions = new ExecutionDataflowBlockOptions { CancellationToken = Cancellation.Token, EnsureOrdered = true };
```
Comment (Naamloos) [inline Obsidian/Client.cs:93]: this uses new ClassName { ...values }

**Review by Naamloos — CHANGES_REQUESTED** [review]
ironically, in this PR alone there still seems to be inconsistencies. Look at the two comments I added. These both have different formats of object initialization.

**Comment by Seb-stian** [comment]
> These both have different formats of object initialization.

Those are different cases though. With `ChatMessage`, type is implicit, thus can be left out. `new { ... }` is not valid (that would be an anonymous type), so `new() { ... }` is used.

**Comment by Naamloos** [comment]
Fair, good point. Will accept and merge o/

**Review by Naamloos — APPROVED** [review]

**Pull request merged**